src/Game.js

```diff
@@ -40,6 +42,50 @@ class Game extends Component {
     return spritesPos;
   }
 
+  /* 
+    Takes username input to save moves
+     ** Contributed by Soham Mondal <contact@sohammondal.com> **
+  */
+  userNameInputForm() {
+    return (
+      <span className="userNameInputForm">
+        <input
+          type="text"
+          placeholder="Your name..."
+          onChange={e => this.setState({ username: e.target.value })}
+        />
+        <button
+          type="submit"
+          onClick={() => this.saveMoves(this.state.username, this.moves)}
+        >
+          Save
+        </button>
+      </span>
+    );
+  }
+
+  /* 
+    Save moves to localStorage 
+     ** Contributed by Soham Mondal <contact@sohammondal.com> **
+  */
+  saveMoves(name, moves) {
+    // console.log(moves);
+    const user = {
+      name,
+      moves
+    };
+    let savedUserMoves = localStorage.getItem("userMoves");
```

Comment:
use single quote

Reply:
I actually have Prettier setup with VS Code which changes all single quotes to double-quotes.